src/security/ldap/src/main/java/org/geoserver/security/ldap/LDAPSecurityProvider.java

```diff
@@ -103,6 +104,13 @@ public GeoServerAuthenticationProvider createAuthenticationProvider(
                 authPopulator =
                         new BindingLdapAuthoritiesPopulator(
                                 ldapContext, ldapConfig.getGroupSearchBase());
+                // set hierarchical configurations
+                BindingLdapAuthoritiesPopulator bindPopulator =
```

Comment:
This is not used anywhere, am I wrong?

Reply:
It is only used as re-typed reference for setting some attributes:
```
BindingLdapAuthoritiesPopulator bindPopulator =
                        (BindingLdapAuthoritiesPopulator) authPopulator;
bindPopulator.setUseNestedParentGroups(ldapConfig.isUseNestedParentGroups());
(...)
```

Reply:
sorry, correct